### PR TITLE
Add example app

### PR DESCRIPTION
**Pull request by lucaswoj**
This PR adds basic project setup and an example app. Run `npm start` from the root directory to build and run the example app. 

Thoughts @asweingarten?

### CODE REVIEW

**Comment by eh-dub** [comment]
bitmoji

Looks good. Quick question, can you summarize when you need to use `$` versus `ObjC`?

**Comment by lucaswoj** [comment]
> Looks good. Quick question, can you summarize when you need to use `$` versus `ObjC`?

`$` is the Objective-C scope. Any variable, class, etc that would be in scope in Objective-C is a property of `$`.

`ObjC` is an object that contains methods allowing manipulation of the Objective-C scope.
